ModuleNotFoundError: No module named 'vine.five'

Installed on Ubuntu with Docker (get same problem on MacOS) 
Celery is installed with version as described in documentation - thought this was a problem with Celery version 

Get error:

api_1        | [2021-01-12 11:32:35 +0000] [1] [INFO] Starting gunicorn 19.9.0
api_1        | [2021-01-12 11:32:35 +0000] [1] [INFO] Listening at: http://0.0.0.0:8000 (1)
api_1        | [2021-01-12 11:32:35 +0000] [1] [INFO] Using worker: sync
api_1        | [2021-01-12 11:32:35 +0000] [8] [INFO] Booting worker with pid: 8
api_1        | [2021-01-12 11:32:35 +0000] [8] [ERROR] Exception in worker process
api_1        | Traceback (most recent call last):
api_1        |   File "/usr/local/lib/python3.7/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
api_1        |     worker.init_process()
api_1        |   File "/usr/local/lib/python3.7/site-packages/gunicorn/workers/base.py", line 129, in init_process
api_1        |     self.load_wsgi()
api_1        |   File "/usr/local/lib/python3.7/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
api_1        |     self.wsgi = self.app.wsgi()
api_1        |   File "/usr/local/lib/python3.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
api_1        |     self.callable = self.load()
api_1        |   File "/usr/local/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
api_1        |     return self.load_wsgiapp()
api_1        |   File "/usr/local/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
api_1        |     return util.import_app(self.app_uri)
api_1        |   File "/usr/local/lib/python3.7/site-packages/gunicorn/util.py", line 350, in import_app
api_1        |     __import__(module)
api_1        |   File "/mutation_maker/server.py", line 19, in <module>
api_1        |     from api import init_api
api_1        |   File "/mutation_maker/api.py", line 25, in <module>
api_1        |     from celery import Celery
api_1        |   File "/usr/local/lib/python3.7/site-packages/celery/__init__.py", line 153, in <module>
api_1        |     from . import local  # noqa
api_1        |   File "/usr/local/lib/python3.7/site-packages/celery/local.py", line 17, in <module>
api_1        |     from .five import PY3, bytes_if_py2, items, string, string_t
api_1        |   File "/usr/local/lib/python3.7/site-packages/celery/five.py", line 7, in <module>
api_1        |     import vine.five
api_1        | ModuleNotFoundError: No module named 'vine.five'
api_1        | [2021-01-12 11:32:35 +0000] [8] [INFO] Worker exiting (pid: 8)
api_1        | [2021-01-12 11:32:35 +0000] [1] [INFO] Shutting down: Master
api_1        | [2021-01-12 11:32:35 +0000] [1] [INFO] Reason: Worker failed to boot.

Doesn't seem to:

Input: % pip show celery

Output:

Name: celery
Version: 4.3.1
Summary: Distributed Task Queue.
Home-page: http://celeryproject.org
Author: Ask Solem
Author-email: ask@celeryproject.org
License: BSD
Location: ~/opt/anaconda3/envs/mutationmaker/lib/python3.7/site-packages
Requires: pytz, vine, billiard, kombu
Required-by: flower


Input: make docker-run

Output:

api_1        | [2021-01-13 09:17:54 +0000] [1] [INFO] Starting gunicorn 19.9.0
api_1        | [2021-01-13 09:17:54 +0000] [1] [INFO] Listening at: http://0.0.0.0:8000 (1)
api_1        | [2021-01-13 09:17:54 +0000] [1] [INFO] Using worker: sync
api_1        | [2021-01-13 09:17:54 +0000] [8] [INFO] Booting worker with pid: 8
api_1        | [2021-01-13 09:17:54 +0000] [8] [ERROR] Exception in worker process
api_1        | Traceback (most recent call last):
api_1        |   File "/usr/local/lib/python3.7/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
api_1        |     worker.init_process()
api_1        |   File "/usr/local/lib/python3.7/site-packages/gunicorn/workers/base.py", line 129, in init_process
api_1        |     self.load_wsgi()
api_1        |   File "/usr/local/lib/python3.7/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
api_1        |     self.wsgi = self.app.wsgi()
api_1        |   File "/usr/local/lib/python3.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
api_1        |     self.callable = self.load()
api_1        |   File "/usr/local/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
api_1        |     return self.load_wsgiapp()
api_1        |   File "/usr/local/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
api_1        |     return util.import_app(self.app_uri)
api_1        |   File "/usr/local/lib/python3.7/site-packages/gunicorn/util.py", line 350, in import_app
api_1        |     __import__(module)
api_1        |   File "/mutation_maker/server.py", line 19, in <module>
api_1        |     from api import init_api
api_1        |   File "/mutation_maker/api.py", line 25, in <module>
api_1        |     from celery import Celery
api_1        |   File "/usr/local/lib/python3.7/site-packages/celery/__init__.py", line 153, in <module>
api_1        |     from . import local  # noqa
api_1        |   File "/usr/local/lib/python3.7/site-packages/celery/local.py", line 17, in <module>
api_1        |     from .five import PY3, bytes_if_py2, items, string, string_t
api_1        |   File "/usr/local/lib/python3.7/site-packages/celery/five.py", line 7, in <module>
api_1        |     import vine.five
api_1        | ModuleNotFoundError: No module named 'vine.five'
api_1        | [2021-01-13 09:17:54 +0000] [8] [INFO] Worker exiting (pid: 8)
api_1        | [2021-01-13 09:17:54 +0000] [1] [INFO] Shutting down: Master
api_1        | [2021-01-13 09:17:54 +0000] [1] [INFO] Reason: Worker failed to boot.
mutation_maker-master_api_1 exited with code 3

--- a/api/requirements.txt
+++ b/api/requirements.txt
@@ -1,4 +1,4 @@
-celery==4.3.0
+celery==4.3.1
 gunicorn==19.9.0
 hug==2.5.1
 redis==3.2.1

--- a/backend/Dockerfile
+++ b/backend/Dockerfile
@@ -4,7 +4,7 @@ LABEL maintainer="Mutation Maker https://github.com/Merck/Mutation_Maker"
 RUN apk add --no-cache \
   build-base==0.5-r1 \
   ca-certificates \
-  curl==7.66.0-r0
+  curl
 
 ENV CELERY_BROKER_URL redis://redis:6379/0
 ENV CELERY_RESULT_BACKEND redis://redis:6379/0

--- a/backend/requirements.txt
+++ b/backend/requirements.txt
@@ -1,7 +1,7 @@
 behave==1.2.6
 biopython==1.73
 boto3==1.13.26
-celery==4.3.0
+celery==4.3.1
 dataclasses==0.6
 flower==0.9.3
 jmespath==0.9.5

